modules/client-configuration/pages/registration-pubcloud.adoc

```diff
@@ -0,0 +1,60 @@
+[[registering.clients.pubcloud]]
+= Register Public Cloud Clients
+
+If you have your clients running in a public cloud instance, there are several methods for registering them to your {productname} Server, in addition to the regular bootstrapping methods.
+For other registration methods, see xref:client-configuration:registration-overview.adoc[].
+
+By default, logging for both the {productname} Server and Salt clients is disabled.
+You can enable logging on Salt clients to provide some additional information during registration.
+Open or create the [path]``/etc/salt/minion.d/log_level.conf`` configuration file on the client, and add or edit this line:
+
+----
+log_level: info
+----
+
+
+
+== Register Salt Clients in the Instance
+
+This method makes the Salt client part of the public cloud image.
```

Comment:
This does not sound right. When you register an instance you do not change the image where it was created from.
Or is this special public cloud terminology that "image" is equal to "instance"?

Reply:
Maybe @mbrookhuis wanted really to modify the image. But this means, you somehow need to "save" the image after you did the changes.

Reply:
Yeah, I was confused about this too?

Reply:
There had been a discussion if it would be needed to include salt-minion in the image. When the image is deployed, a scirpt should be executed (cloud-init) which should build the correct /etc/salt/minion.d/susemanager.conf.  Problem is that this needs to be document for each provider and also I don't know how this works for each provider. If this is to difficult or not doable, mgr-ssh-push-init could be an option. 

I think we need to discuss this with public cloud team. I will ping Don Price. 

Note: for SALT you cannot use the --tunnel option. Also you cannot install salt-ssh. So only usable for normal salt-minion.